app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,36 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="1989 - 2014 Ty Coon"
+LICENSE="GNU GPL v1"
+REVISION="1"
+SOURCE_URI="https://github.com/ewe2/mdate/archive/$portVersion.tar.gz"
+SOURCE_DIR="mdate-$portVersion"
+ARCHITECTURES="x86_64"
+CHECKSUM_SHA256="6d58a8cf63c25b2b4b5b8904ba7c982f18608e8fa65436c555280699a020bcf8"
+PATCHES="mdate-$portVersion.patchset"
+PROVIDES="
+mdate = $portVersion
+cmd:mdate = $portVersion
+"
+REQUIRES="haiku"
+
+BUILD_REQUIRES="haiku_devel"
+BUILD_PREREQUIRES="
+cmd:make
+cmd:gcc
+cmd:ld
+"
+
+BUILD()
+{
+make
+}
+INSTALL()
+{
+ make install
+ mv /usr/bin $binDir
```

Comment:
there's better way to do that.
'''
make install \
    BINDIR=$binDir
    MANDIR=$manDir/man1
'''